package.json

```diff
@@ -33,20 +33,20 @@
     "jsdoc-to-markdown": "^3.0.4",
     "mocha": "^3.5.3",
     "mochainon": "^2.0.0",
-    "mock-fs": "^4.10.4"
+    "mock-fs": "^4.11.0"
   },
   "dependencies": {
     "JSONStream": "^1.3.5",
-    "balena-sdk": "^12.21.0",
-    "balena-settings-client": "^4.0.2",
+    "balena-sdk": "^12.26.7",
+    "balena-settings-client": "^4.0.4",
     "bluebird": "^3.7.2",
-    "chalk": "^1.1.3",
+    "chalk": "^3.0.0",
```

Comment:
None of the major changes affect our usage, it drops support for nodejs 6 but we've already dropped support